packages/yoastseo/src/languageProcessing/helpers/morphology/getAllWordsFromPaper.js

```diff
@@ -24,6 +25,6 @@ export default function( paper ) {
 		altTagsInText.join( " " ),
 	].join( " " );
 
-	return getWords( paperContent ).map(
+	return getContentWordsHelper( paperContent ).map(
```

Comment:
Are you actually using `getAllWordsFromPaper`? As far as I can see, it's only used in the general `getWordForms`, but you're making a specific `getWordForms` for Japanese. If that's the case, I don't think you need to make any modifications in this file.